### PR TITLE
feat(penumbra): add makeDestinationWallets param

### PR DESCRIPTION
**Pull request by codingki**

### CODE REVIEW

File: packages/widget/src/ui/WalletModal/PraxWalletIndex.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <button
+      onClick={(e) => {
+        e.stopPropagation();
+        setEditing(true);
+      }}
+      className="flex flex-row space-x-1 justify-center align-middle text-xs bg-slate-200 rounded-lg p-1.5 px-2"
+    >
+      <p>Address Index {praxWalletIndex}</p>
+      <PencilSquareIcon className="h-4 w-4" />
+    </button>
```
Comment (toddkao) [inline packages/widget/src/ui/WalletModal/PraxWalletIndex.tsx:77]: would be nice to see screenshot of these

Comment (toddkao) [inline packages/widget/src/ui/WalletModal/PraxWalletIndex.tsx:77]: this stuff won't be themed but probably not a big deal?

Comment (codingki) [reply packages/widget/src/ui/WalletModal/PraxWalletIndex.tsx:77]: nice catch, will add StyledThemedButton

Comment (codingki) [reply packages/widget/src/ui/WalletModal/PraxWalletIndex.tsx:77]: https://github.com/user-attachments/assets/30226996-c8b5-4971-b709-6f0904c005c4

**Review by toddkao — APPROVED** [review]

**Pull request merged**